tests/lms/views/error_test.py

```diff
@@ -64,15 +64,6 @@ def test_it_logs_non_http_error_in_sentry(self, pyramid_request):
 
         assert pyramid_request.raven.captureException.call_count == 1
 
-    def test_it_re_raises_exception_in_debug_mode(self, pyramid_request):
-        exc = Exception('test exception msg')
-        pyramid_request.registry.settings = {'debug': True}
-
-        error_views = error.ErrorViews(exc, pyramid_request)
-
-        with pytest.raises(Exception, match="test exception msg"):
-            error_views.error()
```

Comment:
We no longer need to re-raise exceptions in dev because `pyramid_exclog` handles printing exceptions to the terminal for us.